util.go

```diff
@@ -15,3 +15,7 @@ func SnowflakeTimestamp(ID string) (t time.Time, err error) {
 	t = time.Unix(timestamp/1000, 0)
 	return
 }
+
+func MakeIntent(intents Intent) *Intent {
```

Comment:
Put this right next to the enum definition instead.

Doc:

// MakeIntent helps convert a gateway intent value for use in the Identify structure.